types/staking.go

```diff
@@ -6,6 +6,12 @@ const (
 	// default bond denomination
 	DefaultBondDenom = "stake"
 
+	// default maximal voting power ratio
+	DefaultMaxVotingPowerRatio = "0.05"
```

Comment:
does this mean a validator can only have 5% of total delegation?

Reply:
if so, do you think we should set something higher?

Reply:
sg will change it to 10%, also this is the default value at genesis and it can be configured later with a gov proposal